packages/picasso.js/src/core/chart-components/axis/axis-node-builder.js

```diff
@@ -214,18 +230,30 @@ export default function nodeBuilder(isDiscrete) {
     const layered = state.labels.activeMode === 'layered';
     let majorTickNodes;
 
+    if (settings.isRadial) {
+      buildOpts.startAngle = settings.startAngle;
+      buildOpts.endAngle = settings.endAngle;
+      buildOpts.radius = settings.radius;
+    }
     if (settings.line.show) {
       buildOpts.style = settings.line;
       buildOpts.padding = settings.paddingStart;
-
-      nodes.push(buildLine(buildOpts));
+      if (settings.isRadial) {
+        nodes.push(buildArcLine(buildOpts));
+      } else {
+        nodes.push(buildLine(buildOpts));
+      }
     }
     if (settings.ticks.show) {
       buildOpts.style = settings.ticks;
       buildOpts.tickSize = settings.ticks.tickSize;
       buildOpts.padding = tickSpacing(settings);
-
-      majorTickNodes = tickBuilder(major, buildOpts);
+      buildOpts.tickColor = settings.ticks.tickColor || '#000';
```

Comment:
Is this new tickColor setting necessary? 🤔
Where do current ticks (on a non-curved axis) get their color from?

Reply:
The tick stroke comes from settings.ticks.stroke, which can be specified when using picasso. So the new tickColor is not necessary! I fixed it in the last push